### PR TITLE
For Vendors Page

### LINKED ISSUE #442: [TRACKING] Implement new For Vendors page
See wireframes here: https://www.figma.com/file/TfWk1iDHdlt7bSA3DjQuQv/Mobility-Marketplace?type=design&node-id=6921-1040&mode=design&t=R40wIO4T8qpljlNA-0

And tickets here: 

- [x] https://github.com/cal-itp/mobility-marketplace/issues/500
- [x] https://github.com/cal-itp/mobility-marketplace/issues/499
- [x] https://github.com/cal-itp/mobility-marketplace/issues/498
- [x] https://github.com/cal-itp/mobility-marketplace/issues/497
- [x] https://github.com/cal-itp/mobility-marketplace/issues/496
- [x] https://github.com/cal-itp/mobility-marketplace/issues/495

### LINKED ISSUE #495: [vendor page] Add new header image and link to contact us
The goal of this header is to provide an overview of the "story" of MobiMart for a vendor and to provide a clear CTA to contact us, via a button.

Screen Shot 2023-11-28 at 4 54 40 PM

### LINKED ISSUE #496: [vendor page] Add new Focus Areas section
The goal of the Focus Areas section is to help vendors position themselves in what they may be able to offer Cal-ITP but sharing our high-level goals. 

Screen Shot 2023-11-28 at 4 53 35 PM

### LINKED ISSUE #499: [vendor page] Add new Stay up to date section
The new vendor page is designed to provide helpful information to vendors. We should add the existing section we use throughout MobiMart to direct vendors to "stay up to date" by signing up for the newsletter. 

Screen Shot 2023-11-28 at 4 43 30 PM

### LINKED ISSUE #500: [vendor page] Add link to Vendor page from top nav
Screen Shot 2023-11-28 at 4 39 16 PM

The new vendor page includes a requirement to add "For Vendors" as an option from the MobiMart homepage, specifically in the top nav. 

The rationale for this change is that vendors are a significant segment in terms of users of the site and should be able to quickly access information that may be most useful to them. 

**Acceptance Criteria:** 
After this change is implemented, the top nav should read, from left to right: 

- Products 
- Contracts
- For vendors 
- About 
- Contact us

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #442
closes #499
closes #500
closes #496 
closes #495

## What this PR does
- creates new page
- adds newsletter sign up template
- adds page to nav
- add focus areas section
- add header image area

https://deploy-preview-503--cal-itp-mobility-marketplace.netlify.app/for-vendors

### CODE REVIEW

**Comment by machikoyasuda** [comment]
This is the first PR for the For Vendors page - and is now ready for review. All subsequent PRs will go _into_ this PR. 

When this PR is approved, I will *not* merge it in yet. This PR will be merged at the very end, when all the PRs are done.

FYI @thekaveman @angela-tran

**Review by thekaveman — APPROVED** [review]
:mailbox_with_mail:

**Comment by machikoyasuda** [comment]
@mrose914 https://deploy-preview-503--cal-itp-mobility-marketplace.netlify.app/for-vendors is ready for final review

File: src/for-vendors.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        <strong>How can I get a list of partner agencies in California?</strong>
+        <p class="pb-4 mb-md-2">
+          Please visit our <a href="/provider-map">Provider Map page</a> for information about agencies in California. If you have
+          any questions, <a href="/contact">please get in touch</a>.
+        </p>
+
+        <strong>How do I find out more about Cal-ITP?</strong>
+        <p class="pb-4 mb-md-2">
+          You can find more information about Cal-ITP, including resources and recent press, by visiting
+          <a href="/">calitp.org</a>.
```
Comment (thekaveman) [inline src/for-vendors.html:137]: Oops, looks like this `href` is wrong, needs to point to `calitp.org`.

**Review by thekaveman — CHANGES_REQUESTED** [review]
1 tiny fix that I didn't notice from the FAQ PR

**Review by thekaveman — APPROVED** [review]
Approved pending any updates from @mrose914

**Pull request merged**